packages/terra-toolkit-docs/src/terra-dev-site/tool/terra-functional-testing/About.1.tool.mdx

```diff
@@ -196,7 +196,7 @@ it('should hide the caret', () => {
   Terra.hideInputCaret('#inputID');
   const element = browser.element('#inputID');
 
-  expect(element.getCssProperty('caretColor').value).to.equal('rgba(0,0,0,0)');
+  expect(element.getCSSProperty('caretColor').value).to.equal('rgba(0,0,0,0)');
```

Comment:
We'll also want to update the chai `to.equal` to expect's `toEqual` 

```suggestion
  expect(element.getCSSProperty('caretColor').value).toEqual('rgba(0,0,0,0)');
```

Reply:
Updated: https://github.com/cerner/terra-toolkit/pull/589/commits/dc1880f9c4fc5a432c10859b56dd5d749818cfda